Makefile

```diff
@@ -0,0 +1,27 @@
+SOURCES=$(shell ls -1 *.go)
+IMAGE ?= riccardomc/crdbsb
```

Comment:
This doesn't seem very shareable -- I doubt others will want their images tagged with your username.

Reply:
Fixed.